src/commands/cmd_explain.c

```diff
@@ -23,11 +23,9 @@ void Graph_Explain(void *args) {
 	CommandCtx *qctx = (CommandCtx *)args;
```

Comment:
Please rename `qctx`, the leading `q` is misleading, one can think it stands for `query`. please do so for all command handlers

Reply:
done